build/README.md

```diff
@@ -7,13 +7,13 @@ that compiles to JavaScript) in the right order.
 
 The polyPod core code, as well as all the bundled features, are separated into
 several NPM packages that include each other via file URLs. Unfortunately, NPM
-does, at the time of writing, not support building a tree of dependent packages,
+does, at the time of writing, not support building a tree of dependent packages, so
```

Comment:
That'd need some wrapping to still be 80 characters - personally I think the sentence was also understandable before, and it seems pretty unrelated to the rest of this PR.

Reply:
I did it mainly to trigger build. Can revert it, no problem.